libraries/config/src/chain_config.cpp

```diff
@@ -89,7 +91,7 @@ decltype(ChainConfig::predefined_) const ChainConfig::predefined_([] {
     cfg.pbft.lambda_ms_min = 2000;
     cfg.pbft.committee_size = 5;
     cfg.pbft.dag_blocks_size = 100;
-    cfg.pbft.ghost_path_move_back = 1;
+    cfg.pbft.ghost_path_move_back = 0;
```

Comment:
what does this param affect ?

Reply:
It affects count of dag blocks that would be included in pbft block. I will return this back and change only for new tests if it will be needed